Web Application Codes/web/src/components/Error.js

```diff
@@ -0,0 +1,161 @@
+import React, { Component } from 'react'
+
+export class Error extends Component {
+    render() {
+        const navstyle = {
+            backgroundColor: "#EF1D52",
+            height: '10vh',
+        };
+        const bodystyle = {
+            display: 'inline',
+            textAlign: 'center',
+            justifyContent: 'center',
+            color: '#FF4874',
+            alignItems: 'center'
+        }
+        const Logostyle = {
+            widht: '120px !important',
+            display: 'block !important',
+        }
+        return (
+            <div>
+                <center>
+                    <div className='errorNav' style={navstyle}></div>
+
+                    <div className='errorBody' style={bodystyle} >
+                        <div className='topLogo' style={{ width: '40%', margin: 'auto', marginTop: '1%' }} >
+                            <svg className='Path_2076' viewBox='0 0.492 44.69 17.878' >
+                                <path
+                                    id='Path_2076'
+                                    d='M 0.6391686201095581 4.968416690826416 L 5.698694705963135 4.245887279510498 C 6.62128210067749 3.679137229919434 7.578700542449951 3.171795129776001 8.565767288208008 2.726577520370483
```

Comment:
what is all this long cryptic numbers? Are we running quantum algorithms?

Reply:
It's a svg tag for the logo